CHANGELOG.md

```diff
@@ -5,11 +5,29 @@ Please follow indidivudal releases for more information.
 
 Note - Graphback is still not ready to be used in production.
 
+## 0.5.0 (1 August, 2019)
+### Graphback
+#### Features
+- Added ability to choose between methods using config and directives.
+- Added ability for custom Queries and Mutations.
+
+#### Fixes
+- Modularized resolver output into types.
+  
+### Graphback-cli
+#### Fixes
+- Changed resolvers output format in `generate`
```

Comment:
For future: Info about what changed will be useful. Info about changed format is not clear for anyone reading this.